### PR TITLE
propTypes on CurrenyConversion and ES7 Initializers

### PR DESCRIPTION
**Pull request by annamalai-saro**

### CODE REVIEW

File: CurrenyConversion.jsx [hunk 1]
```diff
@@ -4,10 +4,12 @@ es6Promise.polyfill();
...
-  constructor () {
-    super()
-    this.state = { val: 0 }
-  }
+  static propTypes = {
+     from: React.PropTypes.string.isRequired,
+     to: React.PropTypes.string.isRequired,
+     val: React.PropTypes.string.isRequired   
+  };
+  state = { val: 0 };
```
Comment (hemanth) [inline CurrenyConversion.jsx]: ^ needs a `static` here as well and babel needs a flag.

Comment (annamalai-saro) [inline CurrenyConversion.jsx]: @hemanth  its experimental and guess stage 0 in babel ...plus not a confirmed spec I guess ..SORRY !

Comment (annamalai-saro) [inline CurrenyConversion.jsx]: Please change from  ExchangeRate.propTypes to CurrencyConversion.propTypes !

Comment (hemanth) [inline CurrenyConversion.jsx]: Done.

Comment (annamalai-saro) [inline CurrenyConversion.jsx]: :thumbsup:

**Pull request closed without merging**